### PR TITLE
fix(pages): vote user flow

### LINKED ISSUE #55: Refresh votes rows info after click vote (repeat rows are present)

### LINKED ISSUE #61: Don't load user login data in vote page on first loading

### PR DESCRIPTION
**Pull request by AngeloCG97**
### What does this PR do?

- Resolve #55
- Resolve #61

### CODE REVIEW

**Review by xavier506 — APPROVED** [review]
LGTM

File: src/components/Vote/Body/index.tsx [hunk 1]
```diff
@@ -20,13 +19,17 @@ const Body: React.FC<voteBodyProps> = ({
...
   bps
 }) => {
   const classes = useStyles()
 
   useEffect(() => {
+    if (!state?.ual?.accountName) return
+
     loadBps()
-  }, [])
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```
Comment (xavier506) [inline src/components/Vote/Body/index.tsx:31]: do we need the comments?

Comment (tetogomez) [reply src/components/Vote/Body/index.tsx:31]: yes it's for disable eslint rule

**Review by tetogomez — APPROVED** [review]
LGTM

**Pull request merged**